app/services/account_reset/track_irs_event.rb

```diff
@@ -0,0 +1,36 @@
+# Mixin for account reset event tracking
+# Assumes these methods exist on the including class:
+# - sp
+# - success?
```

Comment:
ah I see now that we're using plain `success` not `success?` in those classes. In general, I'd be in favor of switching, but thats not a hard requirement for this PR. might be more accurate to just do:
```suggestion
# - success
```